src/Plugin.InAppBilling.UWP/InAppBillingImplementation.cs

```diff
@@ -1,7 +1,10 @@
 using Plugin.InAppBilling.Abstractions;
 using System;
 using System.Collections.Generic;
+using System.Linq;
 using System.Threading.Tasks;
+using System.Xml;
+using Windows.ApplicationModel.Store;
 
 namespace Plugin.InAppBilling
```

Comment:
I don't really get, what you mean here. UWP needs them both to work with IAPs. How could we remove them? Build works for me here. Any advice?